### DISCUSSION TITLE
Live Preview support

### DISCUSSION THREAD

**Discussion by duncanmcclean** [statamic-rad-pack/runway#446] [Feature Requests]
Runway provides frontend routing for models, like entries.

However, unlike entries, it doesn't provide the "Live Preview" functionality where users can see their content changes in the Control Panel before saving.

Some developers have asked for this feature to be integrated.

**Comment by juliawarnke** [comment]
@duncanmcclean are there plans to implement live preview support in the near future?

**Reply by duncanmcclean** [reply]
Not in the near future, no. Not unless someone's able to submit a PR for it.

**Reply by macaws** [reply]
I can submit a PR @duncanmcclean - actively working on this right now: https://discord.com/channels/489818810157891584/1151898550305837086/1326339672191930368

**Reply by duncanmcclean** [reply]
Awesome, feel free to submit a pull request when you're ready.

**Comment by duncanmcclean** [comment]
Just tagged [v9.0.0-alpha.3](https://github.com/statamic-rad-pack/runway/releases/tag/v9.0.0-alpha.3) which adds support for Live Preview 🚀
